Allow images only collection with CLI (#76)
zegami/zegami-cli: README.md

# Name of the column in the dataset that contains the image name. (optional)
dataset_column: image_name
```

> If `dataset_column` property is not provided, the backend will automatically select the column with the closest match.

Reviewer: ```suggestion
If the `dataset_column` property is not provided, the backend will automatically select the column with the closest match.
```

> If you want to create an images only collection then you do not need to provide `dataset_type` and `path` property.

Reviewer: ```suggestion
To create a collection with only images the `dataset_type` and `path` properties can be omitted.
```
It sounds better to avoid 'you' in this kind of documentation
